subprojects/provider-plugins/src/main/kotlin/org/gradle/kotlin/dsl/provider/plugins/DefaultProjectSchemaProvider.kt

```diff
@@ -125,6 +140,10 @@ private
 val typeOfProject = typeOf<Project>()
 
 
+private
+val typeOfConfigurationContainer = typeOf<NamedDomainObjectContainer<Configuration>>()
```

Comment:
:x: Since we're getting those elements specifically from `project.configurations`, wouldn't it be better to limit the extensions to `ConfigurationContainer` only?

Reply:
We can't because `configurations {}` receiver is `NamedDomainObjectContainerScope<Configuration>`.